<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpecChecker</class>
 <widget class="QWidget" name="SpecChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vLayoutSpecChecker">
   <item>
    <widget class="QGroupBox" name="groupInput">
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="hLayoutInput">
      <item>
       <widget class="QLineEdit" name="leCharacterName">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSearch">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>검색</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClear">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>초기화</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1074</width>
        <height>580</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vLayoutScrollArea">
       <item>
        <layout class="QVBoxLayout" name="vLayoutContents"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
